src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -247,9 +259,9 @@ internal override async ContractTask PostPersistAsync(ApplicationEngine engine)
         {
             // Distribute GAS for committee
 
-            int m = engine.ProtocolSettings.CommitteeMembersCount;
-            int n = engine.ProtocolSettings.ValidatorsCount;
-            int index = (int)(engine.PersistingBlock!.Index % (uint)m);
+            var m = engine.ProtocolSettings.CommitteeMembersCount;
+            var n = engine.ProtocolSettings.ValidatorsCount;
+            var index = (int)(engine.PersistingBlock!.Index % (uint)m);
```

Comment:
It doesn't seem to achieve any functionality; it's just a change in coding style. Frankly, I don't like this coding style. Using the `var` keyword reduces code readability, making it impossible to immediately discern the data type of the variables.

Unless it's something like this:

```c#
var a = new XXX();
```

I can only accept this usage of `var` because the subsequent statements have already explicitly included the variable type.